src/helpers.js

```diff
@@ -1,3 +1,17 @@
+function isNulledBox(box) {
```

Comment:
My second thought about making them availabe was that we can test them. At the moment they are private functions encapsulated in the code. Having them in the SVG obj will make them testable.

I am not sure why it is so bad, to make a camelCase function public api even if other libs have it. Do you think it could fall back on us when we remove our need for that function?
I admit that `matches` does not serve a purpose for the lib. Its a method which you can call on an element already (so its already public). I am completely with you when it comes to helpers which does not bring a benefit but `fullbox` aside I find all our helpers very useful.
However, lets disucss this at another place :)